core/scss/components/card/module.scss

```diff
@@ -1,16 +1,155 @@
 @import '../../tools/functions/core';
-@import '../../settings/rem';
-@import '../../colors/settings';
-@import '../../settings/core';
-@import '../../settings/media';
-@import '../../typography/settings';
-@import '../page-header/settings';
-@import 'settings';
+@import '../../tools/mixins/core';
+
+// Settings
+@import '../../settings/settings';
 
+// Tools
+// @import '../../tools/mixins/background-variant';
```

Comment:
I understand the full list of imports while a WIP, but let's remove all of the commented out imports.